dance/transforms/misc.py

```diff
@@ -117,6 +117,27 @@ def __call__(self, data):
         data.data.raw = data.data.copy()
 
 
+@register_preprocessor("misc")
+class UpdateRaw(BaseTransform):
+    """Update raw data.
```

Comment:
Update raw data.

Reply:
Some data may select genes again after normalizing. At this time, the original raw_data needs to be modified to match the filtered data dimensions.I will add more descriptive information.

Reply:
On the one hand, I think updateRaw also describes the function, although it is not very accurate. On the other hand, the more important reason is that many experimental configuration files have already used updateRaw, which is a bit difficult to modify. I think we can discuss it later.

Reply:
![image](https://github.com/OmicsML/dance/assets/57212168/70dfad04-c82e-4343-83f9-8b14c5f70991)
Many configuration files already use updateRaw, which is a bit troublesome to modify and does not bring much benefit to the project.